<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="64555" started-at="2018-04-26T05:40:15Z" finished-at="2018-04-26T05:41:19Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="64555" started-at="2018-04-26T05:40:15Z" finished-at="2018-04-26T05:41:19Z">
      <class name="com.TestCasesAutomation.TC01_Verify_AddCustomerLink">
        <test-method status="PASS" signature="TC01_Verify_AddCustomerLink_Admin()[pri:0, instance:com.TestCasesAutomation.TC01_Verify_AddCustomerLink@27f674d]" name="TC01_Verify_AddCustomerLink_Admin" duration-ms="22097" started-at="2018-04-26T11:10:17Z" finished-at="2018-04-26T11:10:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC01_Verify_AddCustomerLink_Admin -->
      </class> <!-- com.TestCasesAutomation.TC01_Verify_AddCustomerLink -->
      <class name="com.TestCasesAutomation.TC02_Verify_AddCustomerFiles">
        <test-method status="FAIL" signature="TC02_Verify_Admin_AddCustomer_Fileds()[pri:0, instance:com.TestCasesAutomation.TC02_Verify_AddCustomerFiles@58ceff1]" name="TC02_Verify_Admin_AddCustomer_Fileds" duration-ms="39063" started-at="2018-04-26T11:10:40Z" finished-at="2018-04-26T11:11:19Z">
          <exception class="org.openqa.selenium.remote.UnreachableBrowserException">
            <message>
              <![CDATA[Error communicating with the remote browser. It may have died.Build info: version: '2.53.0', revision: '35ae25b1534ae328c771e0856c93e187490ca824', time: '2016-03-15 10:43:46'System info: host: 'tm6', ip: '192.168.0.43', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_161'Driver info: driver.version: FirefoxDriverCapabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=43.0.1, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]Session ID: ac231c04-3830-4ff3-8518-4eb0b17ff60a]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.remote.UnreachableBrowserException: Error communicating with the remote browser. It may have died.Build info: version: '2.53.0', revision: '35ae25b1534ae328c771e0856c93e187490ca824', time: '2016-03-15 10:43:46'System info: host: 'tm6', ip: '192.168.0.43', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_161'Driver info: driver.version: FirefoxDriverCapabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=43.0.1, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]Session ID: ac231c04-3830-4ff3-8518-4eb0b17ff60aat org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:665)at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:701)at org.openqa.selenium.firefox.FirefoxDriver.getScreenshotAs(FirefoxDriver.java:341)at com.ReportGeneration.GenerateReports.getScreenShot(GenerateReports.java:94)at com.ReportGeneration.GenerateReports.logEvent(GenerateReports.java:68)at com.ScreenFunctionsAutomationTC.Admin_AddCustomer.addCustomerLinkDetails(Admin_AddCustomer.java:246)at com.TestCasesAutomation.TC02_Verify_AddCustomerFiles.TC02_Verify_Admin_AddCustomer_Fileds(TC02_Verify_AddCustomerFiles.java:22)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:73)at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:94)at org.apache.maven.surefire.Surefire.run(Surefire.java:169)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:350)at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1021)Caused by: org.apache.http.conn.HttpHostConnectException: Connect to 127.0.0.1:7056 [/127.0.0.1] failed: Connection refused: connectat org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:151)at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:353)at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:380)at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:236)at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:184)at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:88)at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184)at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:71)at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:55)at org.openqa.selenium.remote.internal.ApacheHttpClient.fallBackExecute(ApacheHttpClient.java:144)at org.openqa.selenium.remote.internal.ApacheHttpClient.execute(ApacheHttpClient.java:90)at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:142)at org.openqa.selenium.firefox.internal.NewProfileExtensionConnection.execute(NewProfileExtensionConnection.java:160)at org.openqa.selenium.firefox.FirefoxDriver$LazyCommandExecutor.execute(FirefoxDriver.java:380)at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:644)... 37 moreCaused by: java.net.ConnectException: Connection refused: connectat java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)at java.net.Socket.connect(Socket.java:589)at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:74)at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:134)... 52 more]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.remote.UnreachableBrowserException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC02_Verify_Admin_AddCustomer_Fileds -->
      </class> <!-- com.TestCasesAutomation.TC02_Verify_AddCustomerFiles -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
